phpBB/develop/adjust_avatars.php

```diff
@@ -131,8 +136,8 @@
 function adjust_avatar($old_name, $midfix)
 {
 	global $config, $phpbb_root_path;
-	
-	$avatar_path = $phpbb_root_path . $config['avatar_path'];
+
+	$avatar_path = $phpbb_root_path . $config['storage\\avatar\\config\\path'];
```

Comment:
I know, there never was a rule about not having (back-)slashes in config names, but without underscores this somehow looks weird to me and breaking the status quo of having configuration names.

Reply:
Actually I it asked on purpose. For me as there is a status quo of not using \ we can reserved it for internal usage like that and have a guarantee of 0 conflict ever.

Reply:
Internal usage means that you're not having any intention to make the path customizable? Then why not mark it as a parameter in a yaml file?

Otherwise I don't understand the term "internal usage" or have a different opinion.

Reply:
the path is customizable from a core page in the acp
but we have a lot of config values related to storage with dynamic names.
Here per example, avatar and path are dynamics the rest is internal. The full name is build internally (including storage, config and \)

Reply:
As personally don't care for this. It's a nice way to prevent conflicts for sure. As long as it doesn't mess up with some dbal

Reply:
Yeah, okay.. It just looks weird to me but other than that it doesn't do anything harmful, I guess.